_content/events/2025-04-21-dsa-backtracking.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Explorando Backtracking: Algoritmo e Aplicações"
+description: "Neste encontro, vamos entender como funciona o backtracking, amplamente utilizado para resolver problemas como geração de permutações, quebra-cabeças, Sudoku e muito mais. Veja como essa técnica permite explorar múltiplas possibilidades de forma estruturada e eficiente."
+date: "2025-04-21"
+time: "20:00-21:30"
+location: "Online via Zoom"
+type: "online"
+recordingLink: "https://www.youtube.com/watch?v=0PYx7erkdXo"
```

Comment:
issue: should be zoom link

Reply:
Great catch!
I have just used a past event as base and... well...